packtools/sps/validation_rules/abstract_rules.json

```diff
@@ -20,7 +20,9 @@
             "article-commentary",
             "book-review",
             "brief-report",
+            "clinical-instruction",
```

Comment:
@Rossi-Luciano este arquivo é sobre abstract, e dentro de `article_type_unexpect` estão os valores de article-type de documentos que não esperam abstract, confere estes valores?

Reply:
@roberta não encontrei referências sobre quais são os valores de `article-type` que não esperam `abstract`. Encontrei os seguintes itens (sem referência) que supostamente não esperam` abstract`, são eles: 

1. `editorial` – Editorial
2. `letter` – Carta ao editor
3. `obituary` – Obituário
4. `correction` – Correção
5. `retraction` – Retração
6. `news` – Notícias
7. `discussion` – Discussão
8. `interview` – Entrevista